src/viewer.js

```diff
@@ -39,24 +45,26 @@ async function startServer(bundleStats, opts) {
 
   if (!chartData) return;
 
-  const app = express();
-
-  // Explicitly using our `ejs` dependency to render templates
-  // Fixes #17
-  app.engine('ejs', require('ejs').renderFile);
-  app.set('view engine', 'ejs');
-  app.set('views', `${projectRoot}/views`);
-  app.use(express.static(`${projectRoot}/public`));
+  const app = polka();
+  app.use(serveStatic(`${projectRoot}/public`));
 
-  app.use('/', (req, res) => {
-    res.render('viewer', {
+  app.get('/', (req, res) => {
+    const data = {
       mode: 'server',
       get chartData() { return JSON.stringify(chartData) },
       defaultSizes: JSON.stringify(defaultSizes)
+    };
+
+    ejs.renderFile('views.ejs', data, ejsOptions, (err, str) => {
+      if (err) {
+        throw err;
```

Comment:
Shouldn't `err` be passed to the `next` route handler?

Reply:
This isn't a middleware but a handler for `GET` and thus doesn't have a `next` handler. I could output an error to the browser but I figured this be enough for the use case of bundle analyzer.

Reply:
With Polka@next, all route handlers follow (req, res, err) signature, so you're no longer limited here.

Here's an [alternative approach](https://github.com/lukeed/polka/issues/73#issuecomment-428315454) using the current `polka@0.5` version